src/style/app.module.css

```diff
@@ -2995,1128 +2641,2457 @@ form > input {
+}
+
+.input {
+  margin-right: 10px;
+}
+
+.borderBox {
+  border: 1px solid #dddddd;
+}
+
+.borderRounded5 {
+  border-radius: 5px;
+}
+
+.borderRounded8 {
+  border-radius: 8px;
+}
+
+.borderRounded24 {
+  border-radius: 24px;
+}
+
+.topRadiusPeople {
+  border-top-left-radius: 24px;
+  border-top-right-radius: 24px;
+}
+
+.bottomRadius {
+  border-bottom-left-radius: 24px;
+  border-bottom-right-radius: 24px;
+}
+
+.shadow {
+  box-shadow: 5px 5px 4px 0px #31bb6b1f;
+}
+
+.colorGreen {
+  color: #31bb6b;
+}
+
+.maxWidthPeople {
+  max-width: 600px;
+}
+
+.mainContainerPeople {
+  width: 50%;
+  flex-grow: 3;
+}
+
+.contentPeople {
+  height: fit-content;
+  min-height: calc(100% - 40px);
+}
+
+.gap {
+  gap: 20px;
+}
+
+.greenBorder {
+  border: 1px solid #31bb6b;
+}
+
+.semiBold {
+  font-weight: 500;
+}
+
+.placeholderColor::placeholder {
+  color: #737373;
+}
+
+.btnsContainer {
+  display: flex;
+  margin: 1.5rem 0;
+}
+
+.btnsContainer .input {
+  flex: 1;
+  min-width: 18rem;
+  position: relative;
+}
+
+.btnsContainer input {
+  outline: 1px solid var(--bs-gray-400);
+  background-color: white;
+}
+
+.btnsContainer .input button {
+  width: 52px;
+}
+
+.titleContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 0.1rem;
+}
+
+.titleContainer h3 {
+  font-size: 1.25rem;
+  font-weight: 750;
+  color: #5e5e5e;
+  margin-top: 0.2rem;
+}
+
+.subContainer span {
+  font-size: 0.9rem;
+  margin-left: 0.5rem;
+  font-weight: lighter;
+  color: #707070;
+}
+
+.progress {
+  display: flex;
+  width: 100%;
+  max-width: 45rem;
+  display: flex;
+  width: 100%;
+  max-width: 45rem;
+}
+
+/* Consolidated styles */
+.progressBar {
+  margin: 0 0.75rem;
+  width: 100%;
+  font-size: 0.9rem;
+  height: 1.25rem;
+  transition: width 0.3s ease-in-out;
+  border-radius: 0.625rem;
+}
+
+/* Data Grid Styles */
+.tableHeaderPledges {
+  background-color: var(--bs-primary);
+  color: var(--bs-white);
+  font-size: 1rem;
+}
+
+.mainContainerPosts {
+  width: 50%;
+  flex-grow: 3;
+  padding: 1rem;
+  max-height: 100%;
+  overflow-y: auto;
+  overflow-x: hidden;
+  background-color: #f2f7ff;
+}
+
+.containerHeightPosts {
+  height: 100vh;
+}
+
+.link {
+  text-decoration: none !important;
+  color: black;
+}
+
+.postInputContainer {
+  margin-top: 0.5rem;
+  margin-bottom: 1rem;
+}
+
+.maxWidthPosts {
+  width: 100%;
+}
+
+.postActionContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: 10px;
+}
+
+.postActionBtn {
+  background-color: white;
+  border: none;
+  color: black;
+}
+
+.postActionBtn:hover {
+  background-color: ghostwhite;
+  border: none;
+  color: black;
+}
+
+.postInput {
+  resize: none;
+  border: none;
+  outline: none;
+  box-shadow: none;
+  background-color: white;
+  margin-bottom: 10px;
+}
+
+.postInput:focus {
+  box-shadow: none;
+}
+
+.postContainer {
+  width: auto;
+  background-color: white;
+  margin-top: 1rem;
+  padding: 1rem;
+  border: 1px solid #dddddd;
```

Comment:
Use a variable. There are other similar colors defined at the top of the file you don't need to redefine one.